apps/percona/chaos/node_freeze/test.yml

```diff
@@ -11,26 +11,21 @@
         - include_tasks: /common/utils/application_liveness_check.yml
           when: liveness_label != ''
 
-         ## RECORD START OF THE TEST IN LITMUS RESULT CR
-        - include_tasks: /common/utils/create_testname.yml
-
```

Comment:
@vibhor can you add this create_testname util

Reply:
@nsathyaseelan done